Dockerfile.image

```diff
@@ -44,6 +44,13 @@ COPY --from=system-agent /usr/bin/rancher-system-agent /usr/sbin/elemental-syste
 # Copy elemental
 COPY --from=elemental-cli /usr/bin/elemental /usr/bin/elemental
 
+# DELETEME once syslinux is added in to the base image or no longer needed
+RUN zypper ar --refresh http://download.opensuse.org/distribution/leap/15.4/repo/oss/ Main && \
+    rpm --import http://download.opensuse.org/distribution/leap/15.3/repo/oss/gpg-pubkey-3dbdc284-53674dd4.asc && \
```

Comment:
For a workaround what is the benefit? Anyway if it's a quick change that's fine.

I'm saying this because this issue is blocking the CI and also manual testing with ISO, so if we could have a fix it could be great :-)

Reply:
mainly just wondering why he choose this over adding our toolkit package, knowing that it was used for the isos before(I think?)

Maybe our package is broken or not valid for this workaround. 

Note that this is just a question, no blocking the PR, I actually approved it ;)

Reply:
oh shit, we have the resolve conversations before merging enabled so that is why this wasnt merged automatically. Sorry about that, didnt thougth we had that enabled anywhere as its a pita. Resolving this comment so it gets merged